examples/GTEx/extract_tile_pw_gtex.py

```diff
@@ -25,18 +26,30 @@ def download_wsi_gtex(dataset_dir: str, sample_ids: List[str]) -> None:
     sample_ids : List[str]
         List of GTEx WSI ids
     """
-    for sample_id in tqdm(sample_ids):
-        if f"{sample_id}.svs" not in os.listdir(dataset_dir):
-
+    dir_files = os.listdir(dataset_dir)
+    sample_ids = set(sample_ids)  # avoid any possible repetition
+    downloaded = set(filter(lambda sid: f"{sid}.svs" in dir_files, sample_ids))
+    to_download = set(sample_ids).difference(downloaded)
+
+    if len(to_download) > 0:
+        if len(downloaded) > 0:
+            print(f"{len(downloaded)} out of {len(sample_ids)} found. Resuming:")
+        else:
+            print(
+                "downloading GTEx WSI dataset. "
+                "This may take several minutes to complete."
```

Comment:
probably much more :)

Reply:
yep.. it took me ages for me yesterday.. that's why I plugged in the "resume" feature. 
As for the message, I thought that "several minutes" generally applies, anyway: If it's a couple of hours, it will be  indeed several minutes 😅